images/converter/converter.go

```diff
@@ -124,3 +142,13 @@ func Convert(ctx context.Context, client Client, dstRef, srcRef string, opts ...
 	}
 	return &res, err
 }
+
+// Convert converts an image in one time.
```

Comment:
Maybe deprecate this in favor of `New().Convert()`

Reply:
Thanks for the reply, removed.